cloud-formation/membership-app.cf.yaml

```diff
@@ -32,10 +32,8 @@ Parameters:
     Description: EC2 instance type
     Type: String
     AllowedValues:
-    - t2.micro
     - t2.small
```

Comment:
Leaving t2.small as an option for now, because it allows us to rollback quickly (without a PR).